pkgs/by-name/re/redfishtool/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, fetchPypi
+, python3
+}:
+
+let
+  pname = "redfishtool";
+  version = "1.1.8";
+in
+python3.pkgs.buildPythonApplication {
+  inherit pname version;
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-X/G6osOHCBidKZG/Y2nmHadifDacJhjBIc7WYrUCPn8=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [ requests python-dateutil ];
+
+  doCheck = false;
```

Comment:
There should probably be a comment explaining the reason why checks are disabled; the package seems to build fine with them enabled, too. Emits a few warnings, though.

Reply:
this was a mistake, i mistook this for another package where tests where flaky. will rectify